encoding/_yaml/stringify_test.ts

```diff
@@ -39,3 +42,93 @@ test: foobar
     assertEquals(stringify(FIXTURE), ASSERTS);
   },
 });
+
+Deno.test({
+  name:
+    "`!!js/*` yaml types are not handled in default schemas while stringifying",
+  fn(): void {
+    const object = { undefined: undefined };
+    assertThrows(
+      () => stringify(object),
+      YAMLError,
+      "unacceptable kind of an object to dump",
+    );
+  },
+});
+
+Deno.test({
+  name:
+    "`!!js/*` yaml types are correctly handled with extended schema while stringifying",
+  fn(): void {
+    const object = {
+      regexp: {
+        simple: /foobar/,
+        modifiers: /foobar/im,
+      },
+      undefined: undefined,
+    };
+
+    const expected = `regexp:
+  simple: !<tag:yaml.org,2002:js/regexp> /foobar/
```

Comment:
so unfortunately, in js-yaml 3 (ported version of Deno), custom tags were dumped with `!<tag:yaml.orf,2022:..>`, and it has been fixed in v4 (https://github.com/nodeca/js-yaml/commit/a0d0caa5aa0f5354fefa9c637cfb7c4c17ef7d02) 

it does works though, it's just the display which is a bit more verbose and less pretty